openwisp_radius/templates/openwisp-radius/admin/rad_batch_users_change_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'admin/change_form.html' %}
+{% load i18n %}
+{% block object-tools-items %}
+{% if download_rad_batch_pdf_url %}
+<li>
+    <a href='{{ download_rad_batch_pdf_url }}' id='downloadpdflink'>{% trans "Download User List PDF" %}</a>
```

Comment:
in the HTML we should use double quotes `"`

I would change the text to `Download user credentials`.

Can you update the user docs to point out to this new button in the relevant page please?